### PR TITLE
Make startup messages less noisy

### PR DESCRIPTION
**Pull request by mwest1066**

### CODE REVIEW

File: workspace_host/interface.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ async.series([
     (callback) => {
         server = http.createServer(app);
         server.listen(workspace_server_settings.port);
-        logger.info(`Listening on port ${workspace_server_settings.port}`);
+        logger.verbose(`Listening on port ${workspace_server_settings.port}`);
```
Comment (jonatanschroeder) [inline workspace_host/interface.js:215]:
```suggestion
        logger.verbose(`Workspace server listening on port ${workspace_server_settings.port}`);
```

**Review by jonatanschroeder — APPROVED** [review]
I don't necessarily have an issue with it (though I agree it's less confusing without the extra port numbers for s3rver and workspaces), but I'm fine with the changes.

**Review by nicknytko — APPROVED** [review]
Looks good — I see the workspace host is littered with `logger.info` calls but we can probably  address those in another PR.

**Comment by mwest1066** [comment]
> Looks good — I see the workspace host is littered with `logger.info` calls but we can probably address those in another PR.

Yep, good point, that will be very noisy for anyone using workspaces locally. We should definitely clean that up at some point.

**Pull request merged**